@types/app.d.ts

```diff
@@ -1,20 +1,25 @@
-import type Block from '@/shared/core/Block';
+import type Block from '@/shared/core';
 
 declare global {
+	export type IObject<T = unknown> = Record<string, T>;
 	export type Nullable<T> = T | null;
```

Comment:
Сюда еще можно добавить undefined. Null и Undefined обозночают одно и то же, разница лишь в том, что undefined задается неявно (хотя можно, конечно, и явно задать)
```suggestion
	export type Nullable<T> = T | null | undefined;
```